src/Presentation/SidebarHtmlBuilder.php

```diff
@@ -32,23 +30,22 @@ public function createHtml( string $searchQuery ): string {
 		$itemType = $query->getItemTypes()[0] ?? null;
 
 		return $this->renderTemplate(
-			$this->buildTabsViewModel(
-				selectedItemType: $itemType
-			),
 			$this->buildFacetsViewModel(
 				itemType: $itemType,
 				query: $query
 			)
 		);
 	}
 
-	private function renderTemplate( array $instancesViewModel, array $facetsViewModel ): string {
+	private function renderTemplate( array $facetsViewModel ): string {
+		if ( count( $facetsViewModel ) === 0 ) {
+			return '';
+		}
```

Comment:
Instead of passing `showSidebar` to the template, rather just not render the template.